src/Basis/Constructors.jl

```diff
@@ -2,28 +2,152 @@
 # finite element bases
 # ------------------------------------------------------------------------------
 
+using Polynomials
+
+# ------------------------------------------------------------------------------
+# conformal map functions for generating transformed polynomial bases
+# ------------------------------------------------------------------------------
+
+"""
+```julia
+sausage(d)
+```
+# Arguments:
+- `d`: polynomial degree of truncated Taylor series expansion of arcsin(s).
+
+# Returns:
+Conformal mapping of Gauss ellipses to sausages using a truncated Taylor expansion of arcsin(s)
+See Figure 4.1 of Hale and Trefethen (2008).
+
+# Example:
+```jldoctest
+# sausage conformal map
+g, derivative(g) = LFAToolkit.sausage(9);
+```
+"""
+function sausage(d)
+    c = zeros(d + 1)
+    c[2:2:end] = [1, cumprod(1:2:d-2) ./ cumprod(2:2:d-1)...] ./ (1:2:d)
+    c /= sum(c)
+    g = Polynomial(c)
+    g_prime = derivative(g)
+    g, g_prime
```

Comment:
I suppose we have the choice here - do we want to keep the dependency on Polynomial or refactor to do this 'by hand'?

Reply:
```suggestion
    c = zeros(d + 1)
    c[2:2:end] = [1, cumprod(1:2:d-2) ./ cumprod(2:2:d-1)...] ./ (1:2:d)
    c /= sum(c)
    g = Polynomial(c)
    g_prime = derivative(g)
    g, g_prime
```
It's trivial to make the coefficients for `g_prime` here, but these get evaluated as functions later and that seems like enough to justify using the library.

Reply:
oh, I see what's going on. Thanks